Sources/Valet/Valet.swift

```diff
@@ -603,6 +629,53 @@ extension Valet {
         return containsObject
     }
 
+    /// Migrates objects matching the input query into the receiving Valet instance.
+    /// - Parameters:
+    ///   - query: The query with which to retrieve existing keychain data via a call to SecItemCopyMatching.
+    ///   - compactMap: A closure that transforms a key:value pair from the raw pair currently in the keychain into a key:value pair we'll insert into the destination Valet. Returning `nil` from this closure will cause that key:value pair not to be migrated.
+    /// - Throws: An error of type `KeychainError` or `MigrationError`.
```

Comment:
Does this get converted over to the `error` parameter description automatically?

Reply:
It does not. Good call out!
```suggestion
    ///   - error: An error of type `KeychainError` or `MigrationError`.
```